app/views/posts/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Detail for post</h1>
+<div class="container">
+  <%= link_to "home", user_path() %>
+  <div class="d-flex flex-column w-50 border p-2">
+    <h3><%= @post.title %> by <%= @post.author.name%> </h3>
+    <div class="d-flex justify-content-end">
+      <span class="">Comments: <%= @post.comments.count%> </span>, 
+      <span class="ml-2" > Comments: <%= @post.likes.count%> </span>
+    </div>
+    <p> <%= @post.text %> </p>
+  </div>
+  <div class="border mt-2 w-50">
+    <h5>Comments</h5>
+    <div class="border mt-1 p-2">
+      <% @post.comments.each do |comment| %>
+        <p>
+          <span> <%= comment.author.name %> : </span>
+          <span> <%= comment.text %> </span>
+        </p>
+      <%end%>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Kindly center the elements to match [the wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png).

  ![image](https://user-images.githubusercontent.com/81468675/230177233-9fc48143-bdfb-4813-a399-50e986a1929a.png)